test/gen-pgbouncer-conf.sh.bats

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bats
```

Comment:
Bats is a Bash-based unit test framework which was introduced in the upstream project.

I installed it on my Mac with `brew install bats` and run the tests locally with `./test/run_all.sh` like so:
```
$ ./test/run_all.sh
 ✓ generates /app/vendor/stunnel/stunnel-pgbouncer.conf
 ✓ generates /app/vendor/pgbouncer/pgbouncer.ini
 ✓ env overrides for pgbouncer low-level network settings
 ✓ returns exit code of 1 with nothing to parse
 ✓ mutates DATABASE_URL
 ✓ substitutes postgres for postgresql in scheme
 ✓ does not mutate other config vars not listed in PGBOUNCER_URLS
 ✓ mutates all config vars listed in PGBOUNCER_URLS

8 tests, 0 failures
```

Also, this coverage gets run by `.travis.yml`.